strings/strings.json

```diff
@@ -450,8 +450,8 @@
     "zh-tw": "搜尋 {0}"
   },
   "SubmitSearch": {
-    "en": "Submit search",
```

Comment:
We already have the fully translated entry for `Search`. Instead, you should go and modify the TypeScript code to reference `l('Search')`  instead of `l('SubmitSearch')`  at the needed place.

Reply:
Alright, I'll change that instead.